cmd/geoipupdate/defaults_notwin.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
+
+package main
+
+// These can be changed using compile time flags. e.g.,
+//
+// go build -ldflags "-X main.defaultConfigFile=/usr/local/etc/GeoIP.conf
+// -X main.defaultDatabaseDirectory=/usr/local/share/GeoIP"
+var (
+	defaultConfigFile        = "/etc/GeoIP.conf"
+	defaultDatabaseDirectory = "/usr/share/GeoIP"
```

Comment:
I wonder if `/usr/local/share/GeoIP` should be the default. That is what you would get if you installed the C version from source.

Reply:
And then we could deal with PPA defaults when building that? That could work and be even closer to the current setup, yeah. I was really focusing on PPA.